### PR TITLE
Refactoring classes

### PR DESCRIPTION
**Pull request by Wikitelmex**
- script.js splited in 4 files
- classes made with SOLID principles aproach
- started styling

### CODE REVIEW

File: script/array.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    const guindex = CreateGuid();
+    const book = {
+      index: guindex,
+      title: addTitle,
+      author: addAuthor,
+    };
+    this.#books.push(book);
+    return book;
+  }
+
```
Comment (DeliceLydia) [inline script/array.js]: - [x] I am not able to add a book in the books array please check your `addBook` method and fix where the issue is
arre

This is the error am getting when I try to add a book

Comment (DeliceLydia) [reply script/array.js]: - [x] It means if I can't add a book I can't test `remove` method please fix your methods

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script/array.js (hunk 1 above)
Comment (Wikitelmex) [reply script/array.js]: was certain uncompatibility in some browsers, it was fixed

**Comment by Davidosky007** [comment]
## Hi, @Wikitelmex @peterdgreat, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
good-job-gif

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**